ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,76 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@st.composite
+def explicit_paddings(draw):
+    x, filters, strides, dilations, data_format = _x_and_filters(draw)
+
+    padding = draw(
+        st.lists(
+            st.tuples(
+                st.integers(min_value=0, max_value=3),
+                st.integers(min_value=0, max_value=3),
+            ),
+            min_size=2,
+            max_size=2,
+        )
+    )
+
+    if data_format == 'NCHW':
+        padding.append((0, 0))
+    else:
+        padding.insert((0, 0), (0, 0))
+
```

Comment:
You wrote above that explicit padding should be [[0, 0], [pad_top, pad_bottom], [pad_left, pad_right], [0, 0]] when the data_format is "NHWC", and [[0, 0], [0, 0], [pad_top, pad_bottom], [pad_left, pad_right]] when the data_format is "NCHW". I think you're still missing a part.

Reply:
oh sorry sill mistake, this should be fine I think so
```

    if data_format == "NHWC":
        padding = [(0, 0), (padding[0][0], padding[0][1]), (padding[1][0], padding[1][1]), (0, 0)]
    else:
        padding = [(0, 0), (0, 0), (padding[0][0], padding[0][1]), (padding[1][0], padding[1][1])]

    if data_format == "NCHW":
        padding.insert(0, (0, 0))
        padding.insert(1, (0, 0))
```

Reply:
(padding[0][0], padding[0][1]) is just padding[0]. You could simplify.
Also, I don't think you need this part anymore
```
    if data_format == "NCHW":
        padding.insert(0, (0, 0))
        padding.insert(1, (0, 0))
```